code/src/Albums.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Icons } from "./Icons";
+
+export const Albums = props => {
+  return (
+    <div
+      className="AlbumContainer"
+      style={{ backgroundImage: `url(${props.image})` }}
+    >
+      <div className="ArtistContainer">
+        <div>
+          <div className="iconsContainer">
```

Comment:
You have a lot of divs and containers here (and in general).. There's nothing specifically wrong with that, but in future projects try to challenge yourself to use fewer divs like this - you can often get away with using very few elements